defaults/main.yml

```diff
@@ -90,7 +90,10 @@ nextcloud_preview_app_jpeg_quality: "60"
 # - `nextcloud_container_image_customizations_dockerfile_body_custom`
 # - `nextcloud_container_image_customized`
 # - `nextcloud_container_image_final`
-nextcloud_container_image_customizations_enabled: "{{ nextcloud_container_image_customizations_php_imageick_installation_enabled }}"
+nextcloud_container_image_customizations_enabled: >
+  "{{ nextcloud_container_image_customizations_php_imageick_installation_enabled 
```

Comment:
Not sure if `>` + `"` is the right thing here.

I think it's better to go with something like:

```yml
nextcloud_container_image_customizations_enabled: |
  {{
    (
      nextcloud_container_image_customizations_php_imageick_installation_enabled
      or
      nextcloud_container_image_customizations_samba_enabled
    )
  }}
```

Reply:
totally grateful to be learning best practices! when i looked up yml multiline syntax the ref i saw indicated ">" was the default, perhaps to avoid any complications where newlines are interpreted confusingly? but i definitely have seen a lot of "|" so probably that's more standard? i do wonder if the indentation is crucial here, still figuring that out... thanks for the help!